### PR TITLE
Update vignette

### PR DESCRIPTION
**Pull request by Robinlovelace**
Ready for xmas release

### CODE REVIEW

File: vignettes/introducing-stplanr.Rmd [hunk 1]
```diff
@@ -14,40 +14,181 @@ bibliography: references.bib
...
+
+Load the package as follows:
+
+```{r}
+library(stplanr)
+```
+
+# Links to related packages
+
+The message printed when **stplanr** is loaded tells us that it also loads **sp**. This means that **stplanr** *depends* on on **sp**.
```
Comment (richardellison) [inline vignettes/introducing-stplanr.Rmd]: "*depends* on on **sp**" should be "*depends* on **sp**".

**Comment by richardellison** [comment]
The introductory vignette is looking good. The build is failing though because we need to install the OpenStreetMap package for one of the examples. Not sure what this means for CRAN submission?

**Comment by Robinlovelace** [comment]
I think it can work as a suggests... Or I could just save the image. Prefer a suggest though - will try now.

**Comment by richardellison** [comment]
I agree a suggest would be better if it works. Seems like OpenStreetMap needs rJava so the travis configuration will need to be updated to install java.

**Comment by Robinlovelace** [comment]
Thanks for the heads up - hope that fixes it... Will fix typo(s) in meantime.

**Comment by richardellison** [comment]
Works now. Ready to merge.

**Pull request merged**